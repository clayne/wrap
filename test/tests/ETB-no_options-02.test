wrapc | /dev/null | | ETB-02.txt | 0
