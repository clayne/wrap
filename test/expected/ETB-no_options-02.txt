    // don't emit leader_tws for blank lines
    bool const is_blank_line = strcmp( pbuf, "\n" ) == 0;
    FPRINTF( fout, "%s%s%s", leader, is_blank_line ? "" : leader_tws, pbuf );
